src/main/java/io/cdap/plugin/gcp/gcs/actions/GCSBucketDelete.java

```diff
@@ -119,13 +128,77 @@ public void run(ActionContext context) throws Exception {
       if (fs.exists(gcsPath)) {
         try {
           fs.delete(gcsPath, true);
+          deleteCount = deleteCount + 1;
         } catch (IOException e) {
           LOG.warn(
             String.format("Failed to delete path '%s'", gcsPath)
           );
         }
       }
     }
+
+    for (GCSPath gcsPath : gcsPathsWild) {
+      String gcsPathStr = gcsPath.getUri().toString();
```

Comment:
Can you group these stuff under the for loop into a function and write unit tests against the function. It's a little difficult to visualize the input/output.

Reply:
Yeah, let me try to use the function